CHANGELOG.md

```diff
@@ -1,27 +1,34 @@
 # Main (unreleased)
 
+# v0.15.0 (2021-06-03)
+
 BREAKING CHANGE: Configuration of Tempo Autologging changed in this release.
 Please review the [migration
 guide](./docs/migration-guide.md) for details.
 
-- [FEATURE] Add support for exemplars (@mapno)
+- [FEATURE] Add support for exemplars. (@mapno)
 
 - [ENHANCEMENT] Add the option to log to stdout instead of a Loki instance. (@joe-elliott)
 
 - [ENHANCEMENT] Running the Agent as a DaemonSet with host_filter and role: pod
   should no longer cause unnecessary load against the Kubernetes SD API.
   (@rfratto)
 
-- [ENHANCEMENT] Update Prometheus to v2.27.0, Loki to d88f3996eaa2 and Cortex to d382e1d80eaf.
-  Loki and Cortex are non-release builds, and were needed to support exemplars. (@mapno)
+- [ENHANCEMENT] Update Prometheus to v2.27.0.(@mapno)
+
+- [ENHANCEMENT] Update Loki dependency to d88f3996eaa2. This is a non-release
+  build, and was needed to support exemplars. (@mapno)
+
+- [ENHANCEMENT] Update Cortex dependency to to d382e1d80eaf. This is a
+  non-release build, and was needed to support exemplars. (@mapno)
```

Comment:
I split these up into three changes since IMO they're separate updates, despite being a part of one PR